book/website/collaboration/academic-industry/academic-industry-community.md

```diff
@@ -3,7 +3,18 @@
 
 Building an engaged community around any collaboration can be key for long term success, sustainability and legacy. 
 For academic-industry collaborations, a focus on community can be crucial as often the two organisations have not worked together previously. 
-The Turing-Roche partnership hired a full time Community Manager for this purpose; helping to navigate any differences in ways of working, facilitating initial and longer term connections and creating a sense of ownership and engagement around the partnership. 
+
+## Challenges of Building an Academia-Industry Community
```

Comment:
I feel that if we put the community building challenges in this file, readers will be confused about where to look for the comprehensive list of challenges in community building (as right now it is written as "some to consider". I wonder if it might be worth adding a new file, titled maybe something like "curated best practices for industry academia collaboration", where we can continuously grow the list of best practices and challenges in industry-academia collab, so that everyone has a specific go-to-space if they want to look for them. I also think every best practice/challenge should be tied to at least one piece of evidence, whether it to a specific case study from the library of case studies, or from an online resource. Either way, a link/reference should be provided so that everything is as evidence-based as possible